Interfaces/ITokenAcquisition.cs

```diff
@@ -69,7 +69,7 @@ public interface ITokenAcquisition
         /// };
         /// </code>
         /// </example>
-        void AddAccountToCacheFromJwt(JwtBearer.TokenValidatedContext tokenValidationContext, IEnumerable<string> scopes = null);
```

Comment:
Warning, there are 2 TokenValidatedContext 
a JwtBearer one and an OIDCOne
I would prefer that this is explicit? is it an issue?

Reply:
Valid point, agree.